src/utils/metadata.ts

```diff
@@ -99,6 +99,22 @@ function getAlgorithmFileExtension(fileUrl: string): string {
   return splitedFileUrl[splitedFileUrl.length - 1]
 }
 
+function sanitizeUrl(url: string): string {
+  return url.replace('javascript:', '')
+}
+
+function transformFiles(files: string | File[]): File[] {
+  return (
+    typeof files !== 'string' &&
+    files?.length && [
+      {
+        ...(files as File[])[0],
+        url: sanitizeUrl((files as File[])[0].url)
+      }
+    ]
+  )
+}
```

Comment:
suggestion(non-blocking): This is just a simple check and unnecessarily difficult to read. Extracting the object in the array to a variable might improve readability here.